CONTRIBUTING.md

```diff
@@ -1,25 +1,20 @@
 # Contributing Guidelines
 
-Thank you for your interest in contributing to our project. Whether it's a bug report, new feature, correction, or additional
-documentation, we greatly value feedback and contributions from our community.
-
-Please read through this document before submitting any issues or pull requests to ensure we have all the necessary
-information to effectively respond to your bug report or contribution.
+Thank you for your interest in contributing to our project. Whether it's a bug report, new feature, correction, or additional documentation, we greatly value feedback and contributions from our community.
 
+Please read through this document before submitting any issues or pull requests to ensure we have all the necessary information to effectively respond to your bug report or contribution.
```

Comment:
I don't see anywhere that you changed the above content? Other than removing the newline? Or am I missing something?

Maybe you edited the files via GitHub instead of an editor or something?

Reply:
fixed lines and spacing here.